src/global/text.scss

```diff
@@ -152,7 +152,7 @@ h1.title {
   font-family: var(--text-light-weighted-font-family, var(--bloom-font-alt-sans));
   font-weight: var(--text-light-weighted-font-weight, 600);
   color: var(--text-light-weighted-color, var(--bloom-text-color-light));
-  font-size: var(--text-light-weighted-font-size, var(--bloom-font-size-xs));
+  font-size: var(--text-light-weighted-font-size, var(--bloom-font-size-sm));
```

Comment:
This is a global text style change but this style is only used in this component